src/elements/List/List.js

```diff
@@ -52,6 +62,7 @@ List._meta = {
   props: {
     size: sui.sizes,
     aligned: sui.verticalAlignments,
+    relaxed: 'very',
```

Comment:
Just a fix from an unrelated previous omission. It can either be a string (i.e. very) or a plain Boolean.